### PR TITLE
Run in-place pod resize node tests in parallel

### PR DESCRIPTION
**Pull request by AnishShah**
/hold
until https://github.com/kubernetes/kubernetes/pull/128719 is merged

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
ping when ok to merge

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -814,7 +814,7 @@ presubmits:
             - --node-tests=true
             - --provider=gce
             - --runtime-config=api/all=true
-            - --test_args=--nodes=1 --skip="" --focus="\[Feature:InPlacePodVerticalScaling\]"
+            - --test_args=--nodes=8 --skip="" --focus="\[Feature:InPlacePodVerticalScaling\]"
```
Comment (hshiina) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:817]: This feature label was deleted when this feature was promoted to beta (#128682).
```suggestion
            - --test_args=--nodes=8 --skip="" --focus="Pod InPlace Resize Container"
```

Comment (AnishShah) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:817]: Reverting this suggestion after https://github.com/kubernetes/kubernetes/pull/128875

**Comment by tallclair** [comment]
/hold cancel

https://github.com/kubernetes/kubernetes/pull/128719 merged

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**